mlflow/protos/model_registry.proto

```diff
@@ -224,6 +224,78 @@ service ModelRegistryService {
       rpc_doc_title: "Get Download URI For ModelVersion Artifacts",
     };
   }
+
+  // .. note::
+  //     Experimental: This API may change or be removed in a future release without warning.
+  rpc setRegisteredModelTag (SetRegisteredModelTag) returns (SetRegisteredModelTag.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "POST",
+        path: "/preview/mlflow/registered-models/set-tag"
+        since { major: 2, minor: 0},
+      }, {
+        method: "POST",
+        path: "/mlflow/registered-models/set-tag"
+        since { major: 2, minor: 0},
```

Comment:
remove the non-preview paths, from all new endpoints

Reply:
Looks like all other MR APIs still use `/preview`, and the feature is technically still in a preview/experimental phase. Is there some context here I'm missing?